api/registry/management/commands/get_unmonitored_urls_config.py

```diff
@@ -0,0 +1,148 @@
+                },
+                ("GET", "/ceramic-cache/weights"): {
+                    "url": f"{base_url}ceramic-cache/weights",
+                },
+                # Auth required
+                ("GET", "/ceramic-cache/stake/gtc"): {
+                    "url": f"{base_url}ceramic-cache/stake/gtc",
+                    "http_headers": {
+                        "Authorization": f"Bearer {CERAMIC_CACHE_JWT_TOKEN}"
+                    },
+                },
+                ("GET", "/ceramic-cache/score/{address}"): {
+                    "url": f"{base_url}ceramic-cache/score/{CERAMIC_CACHE_ADDRESS}",
+                    "http_headers": {
+                        "Authorization": f"Bearer {CERAMIC_CACHE_JWT_TOKEN}"
+                    },
+                },
+                ("GET", "/ceramic-cache/tos/accepted/{tos_type}/{address}"): {
+                    "url": f"{base_url}ceramic-cache/tos/accepted/ISI/{CERAMIC_CACHE_ADDRESS}",
+                    "http_headers": {
+                        "Authorization": f"Bearer {CERAMIC_CACHE_JWT_TOKEN}"
+                    },
+                },
+                ("GET", "/ceramic-cache/tos/message-to-sign/{tos_type}/{address}"): {
+                    "url": f"{base_url}ceramic-cache/tos/message-to-sign/IST/{CERAMIC_CACHE_ADDRESS}",
+                    "http_headers": {
+                        "Authorization": f"Bearer {CERAMIC_CACHE_JWT_TOKEN}"
+                    },
+                },
+                ("POST", "/ceramic-cache/stamps/bulk"): {
+                    "url": f"{base_url}ceramic-cache/stamps/bulk",
+                    "http_headers": {
+                        "Authorization": f"Bearer {CERAMIC_CACHE_JWT_TOKEN}"
+                    },
+                    "payload": stamps,
+                },
+                ("PATCH", "/ceramic-cache/stamps/bulk/meta/compose-db"): {
+                    "url": f"{base_url}ceramic-cache/stamps/bulk/meta/compose-db",
+                    "http_headers": {
+                        "Authorization": f"Bearer {CERAMIC_CACHE_JWT_TOKEN}"
+                    },
+                    "payload": [],
+                },
+                ("POST", "/ceramic-cache/score/{address}"): {
+                    # TODO: do we still use this POST API ???
+                    "skip": True,  # Skipping because uptime robot api rejects creating multiple monitors on same endpoint
+                    "url": f"{base_url}ceramic-cache/score/{CERAMIC_CACHE_ADDRESS}",
+                    "http_headers": {
+                        "Authorization": f"Bearer {CERAMIC_CACHE_JWT_TOKEN}"
+                    },
+                    "payload": {"alternate_scorer_id": REGISTRY_SCORER_ID},
+                },
+                ("PATCH", "/ceramic-cache/stamps/bulk"): {
+                    "skip": True,  # Skipping because uptime robot api rejects creating multiple monitors on same endpoint
+                },
+                ("DELETE", "/ceramic-cache/stamps/bulk"): {
```

Comment:
Wondering if it would be more effective to flag this as not created instead of skipping. That way whoever makes the changes can go into uptime robot and manually create the monitor. That way everything is covered